components/brave_wallet/common/brave_wallet_response_helpers.h

```diff
@@ -21,6 +21,9 @@ std::unique_ptr<base::Value> GetProviderRequestReturnFromEthJsonResponse(
 std::unique_ptr<base::Value> GetProviderErrorDictionary(
     mojom::ProviderError code,
     const std::string& message);
+std::unique_ptr<base::Value> GetProviderErrorDictionary(
```

Comment:
```suggestion
std::unique_ptr<base::Value> GetSolanaProviderErrorDictionary(
```

Since this is using a Solana specific Provider error should the name of this function also specify that it's solana specific? Not certain of the common code conventions here so this is partially just for my learning.

Reply:
I will rename it as you suggested, thanks

Reply:
fixed in https://github.com/brave/brave-core/pull/12554/commits/fd50ea99aee024d1132b16510bb6fa7d77154c84